packages/docs/src/components/page-configs/ui-elements/va-data-table/api-options.ts

```diff
@@ -0,0 +1,36 @@
+import { defineManualApi } from '../../../DocsApi/ManualApiOptions'
+
```

Comment:
Can you add a bit of text here? Looks pretty confusing to me.

Reply:
I added a bit explain text to each example